.travis.yml

```diff
@@ -31,6 +35,9 @@ jobs:
   fast_finish: true
 
 before_install:
+  - if [[ $APP_ENV = 'elasticsearch' ]]; then
+      curl -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.5.0.deb && sudo dpkg -i --force-confnew elasticsearch-6.5.0.deb && sudo service elasticsearch restart;
```

Comment:
From SymfonyInsight:
> Unable to parse at line 39 (near " curl -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.5.0.deb && sudo dpkg -i --force-confnew elasticsearch-6.5.0.deb && sudo service elasticsearch restart;").

Any idea @api-platform/core-team?

Reply:
Ignore it. Insight isn’t good at parsing Travis files (Travis’s YAML parser is very tolerant)

Reply:
It's valid YAML. SymfonyInsight needs to fix their parser (Symfony YAML component?)

Reply:
I fixed it by creating a `.symfony.insight.yaml` file and by adding an exclude pattern.

Reply:
I don't think we should do that...

Reply:
I'm agree but since I don't have the rights to ignore the error, it was a quick way to do it...

Reply:
Did you try quoting the url? The parser might be confused by the `:` and trying to treat it as a key? Lol... (I'm not sure...)

Reply:
I'll ignore the error later, don't worry about that.

Reply:
> Did you try quoting the url?

It doesn't work.

The revert is done, you can ignore the error.